### ISSUE TITLE
less_equal does not work w/o  numpy fallback

### ISSUE THREAD

**Issue by DenisScherbakov** [IntelPython/dpnp#1251]
Example test code from [official doc](https://intelpython.github.io/dpnp/reference/generated/dpnp.less_equal.html#dpnp-less-equal)
does not work:
```
import dpnp as np
x1 = np.array([4, 2, 1])
x2 = np.array([2, 2, 2])
out = np.less_equal(x1, x2)
[i for i in out]
```

error is:
```
File "test.py", line 4, in <module>
    out = np.less_equal(x1, x2)
  File "..../lib/python3.9/site-packages/dpnp/dpnp_iface_logic.py", line 620, in less_equal
    return call_origin(numpy.less_equal, x1, x2)
  File "dpnp/dpnp_utils/dpnp_algo_utils.pyx", line 120, in dpnp.dpnp_utils.dpnp_algo_utils.call_origin
NotImplementedError: Requested funtion=less_equal with args=(<dpnp.dpnp_array.dpnp_array object at 0x7f5ce86b29d0>, <dpnp.dpnp_array.dpnp_array object at 0x7f5ce86b2d60>) and kwargs={} isn't currently supported and would fall back on NumPy implementation. Define enviroment variable `DPNP_RAISE_EXCEPION_ON_NUMPY_FALLBACK` to `0` if the fall back is required to be supported without rasing an exception.
```

**Comment by antonwolfy** [comment]
@DenisScherbakov, the fix is merged in #1275.

**Comment by DenisScherbakov** [comment]
Issue is not reproduced with provided fix.
